### DISCUSSION TITLE
Changing FlowiseAI Name with a custom name

### DISCUSSION THREAD

**Discussion by Eng-AliKazemi** [FlowiseAI/Flowise#2432] [Q&A]
Hi everyone,

I want to express my gratitude to the contributors for creating such a valuable open-source tool.

Can you help me with the process of rebranding FlowiseAI to a new brand?

I am specifically interested in locating the main files that contain the name and logo.

**Comment by VSM97** [comment]
Poke around the packages/ui directory.

**Reply by Eng-AliKazemi** [reply]
thanks
